SearchService2/setup.py

```diff
@@ -34,5 +34,8 @@
   include_package_data=True,
   entry_points={'console_scripts': [
     'appscale-search2=appscale.search.search_server:main',
+    'appscale-search2-reindex=appscale.search.scripts:reindex',
+    'list-solr-collections=appscale.search.scripts:list_solr_collections',
+    'delete-solr-collection=appscale.search.scripts:delete_solr_collection',
   ]}
```

Comment:
Commands we own should probably start with `appscale-`

Reply:
Thanks for the review!
I see that it breaks our usual commands style. But I think it is acceptable as the package should be installed in venv, so these binaries won't be in a global namespace.

Though I don't have a strong opinion on this.. would you still prefer to rename it?

Reply:
I think it is preferable to use consistent  / discoverable names where possible. I also think it is fine to merge without changes, this is just nice to have.